starport_template/lib/pages/sign_transaction_page.dart

```diff
@@ -58,7 +62,30 @@ class SignTransactionPage extends StatelessWidget {
               child: CosmosElevatedButton(
                 text: 'Tap to sign',
                 prefixIcon: Image.asset('assets/images/face_id.png'),
-                onTap: () {},
+                onTap: () async {
+                  StarportApp.walletsStore.sendTokens(
+                    info: StarportApp.walletsStore.selectedWallet,
+                    balance: Balance(
+                      amount: transaction.amount,
+                      denom: balance.denom,
+                    ),
+                    toAddress: transaction.recipient,
+                    password: '',
```

Comment:
add a "TODO" next to password stating that 
```dart
//TODO create separate method that will use empty password for biometric or ask the user for one otherwise
```